configure

```diff
@@ -2021,9 +2021,9 @@ $as_echo "$as_me: WARNING: $2: see the Autoconf documentation" >&2;}
 $as_echo "$as_me: WARNING: $2:     section \"Present But Cannot Be Compiled\"" >&2;}
     { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: $2: proceeding with the compiler's result" >&5
 $as_echo "$as_me: WARNING: $2: proceeding with the compiler's result" >&2;}
-( $as_echo "## ------------------------------------ ##
```

Comment:
`configure` need generate by `configure.ac` in stand build env...

why i have not saw the line changed in `configure.ac`?

Reply:
Aha, this file is actually generated by `autoconf 2.69`. 
As for your questions, I cannot answer them exactly...